packages/vaadin-grid/src/vaadin-grid.js

```diff
@@ -6,10 +6,10 @@
 import { html } from '@polymer/polymer/lib/utils/html-tag.js';
 import { beforeNextRender } from '@polymer/polymer/lib/utils/render-status.js';
 import { Debouncer } from '@polymer/polymer/lib/utils/debounce.js';
-import { timeOut, animationFrame } from '@polymer/polymer/lib/utils/async.js';
+import { animationFrame } from '@polymer/polymer/lib/utils/async.js';
 import { ElementMixin } from '@vaadin/vaadin-element-mixin/vaadin-element-mixin.js';
 import { ThemableMixin } from '@vaadin/vaadin-themable-mixin/vaadin-themable-mixin.js';
-import { ScrollerElement } from './vaadin-grid-scroller.js';
+import { PolymerElement } from '@polymer/polymer/polymer-element.js';
```

Comment:
nit: let's combine this with the top-level import for `html` tag like this:

```suggestion
import { PolymerElement, html } from '@polymer/polymer/polymer-element.js';
```

Reply:
Done